tools/library/scripts/manifest.py

```diff
@@ -352,17 +355,17 @@ def append(self, operation):
       self.operations[operation.operation_kind][configuration_name].append(operation)
       self.operation_count += 1
     else:
-      print("Culled {} from manifest".format(operation.procedural_name()))
+      _LOGGER.debug("Culled {} from manifest".format(operation.procedural_name()))
```

Comment:
Only change here. Rest are whitespace normalization.

Reply:
Hi! The file this outputs to is already intended to be a log file. Is this something that gets in your workflow? We intended these to be logged because Hopper kernels won't generate without 90a as the arch and we wanted to provide info to the user which kernels are not getting generated etc.

Reply:
Ah I think this would be an issue if someone runs this script 
outside of the CMake build too to create custom manifests. Fair

Reply:
Yeah, we (AITemplate) run this out of CMake so would be good to provide a way to turn this off.

Reply:
Agreed! Sorry for not foreseeing this usecase